LICENSE.md

```diff
@@ -70,9 +71,9 @@ their own licenses:
 
 The following components of Julia's standard library have separate licenses:
 
-- base/fftw.jl (see [FFTW](http://fftw.org/doc/License-and-Copyright.html))
```

Comment:
probably covered by the "Julia's standard library uses the following external libraries" bullet above, same as for the rest of the ccall dependencies